cmd/bucket-handlers.go

```diff
@@ -858,10 +858,7 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 	globalNotificationSys.LoadBucketMetadata(GlobalContext, bucket)
 
 	// Call site replication hook
-	if err := globalSiteReplicationSys.MakeBucketHook(ctx, bucket, opts); err != nil {
-		writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL)
-		return
-	}
+	go globalSiteReplicationSys.MakeBucketHook(ctx, bucket, opts)
```

Comment:
We can't do uncontrolled goroutines here.. IMO we should do what we did before just don't have to return an error.

Reply:
fixed